ChangeLog.d/fix_aead_libriver1_build.txt

```diff
@@ -0,0 +1,3 @@
+Bugfix
+   * Fix bugs and enable testing of AEAD
```

Comment:
I think that might be a bit confusing for the average user. I don't think we need to mention details of our testing strategy such as libtestdriver1. Also "fix bugs" is a bit vague about what the bugs where. Perhaps something more suitable would be "Fix compilation errors when trying to build with PSA drivers for AEAD (GCM, CCM, Chacha20-Poly1305)." (I think we only need to mention the worst impact: who cares about compiler warning when you also has compiler errors?)